@here/harp-text-canvas/lib/rendering/TextStyle.ts

```diff
@@ -447,6 +472,28 @@ export class TextLayoutStyle {
         this.m_params.horizontalAlignment = value;
     }
 
+    /**
+     * Text element horizontal position relating to the placement point.
+     */
+    get verticalPlacement(): VerticalPlacement {
+        return (this.m_params.verticalAlignment! as unknown) as VerticalPlacement;
+    }
+    set verticalPlacement(value: VerticalPlacement) {
+        // Update vertical alignment, both values works interchangeably.
+        this.m_params.verticalAlignment = (value as unknown) as VerticalAlignment;
+    }
+
+    /**
+     * Text element vertical position relating to the point of placement.
```

Comment:
And if you consider not removing this, add note that `*placement` overweites `*alignment`